src/main/java/com/webservices/company/repository/EmployeeRepository.java

```diff
@@ -1,5 +1,103 @@
 package com.webservices.company.repository;
 
-public class EmployeeRepository {
+import com.webservices.company.domain.Company;
+import com.webservices.company.domain.Employee;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementCreator;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.stereotype.Component;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.Timestamp;
+import java.util.List;
+
+@Component
+public class EmployeeRepository implements IEmployeeRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    private final RowMapper<Employee> rowMapper;
```

Comment:
I see that in some cases class attributes are split by lines, some times now. Suggest using 1 approach